sources/osgViewer/Viewer.js

```diff
@@ -436,11 +436,9 @@ define( [
             var deltaS = Timer.instance().deltaS( this._startTick, Timer.instance().tick() );
             frameStamp.setReferenceTime( deltaS );
 
-            // reference time
-            if ( sTime === Number.MAX_VALUE )
-                frameStamp.setSimulationTime( frameStamp.getReferenceTime() );
-            else
-                frameStamp.setSimulationTime( sTime );
+            var lastSimul = frameStamp.getSimulationTime();
+            frameStamp.setSimulationTime( sTime === Number.MAX_VALUE ? deltaS : sTime ); // set simul time
+            frameStamp.setDeltaTime( frameStamp.getSimulationTime() - lastSimul ); // compute delta since last tick
 
             var deltaFrameTime = frameStamp.getReferenceTime() - previousReferenceTime;
             this.getViewerStats().setAttribute( previousFrameNumber, 'Frame rate', 1.0 / deltaFrameTime );
```

Comment:
Divide by 0 ?
Like user not using requestAnimFrame and/or submitting multiple render per frame 
And with a very fast cpu/gpu +  bad CPU precision timer